apps/citizen-func/package.json

```diff
@@ -1,61 +1,45 @@
 {
   "name": "citizen-func",
   "version": "5.5.4",
-  "license": "MIT",
+  "license": "UNLICENSED",
   "scripts": {
-    "prebuild": "yarn openapi:bundle && rimraf ./generated && yarn generate:definitions",
-    "build": "tsc",
-    "test": "jest",
-    "test:coverage": "jest --coverage",
+    "build": "yarn openapi:bundle && rimraf ./generated && yarn generate:definitions && tsc",
```

Comment:
@lucacavallaro I understand that this is intended to maintain the same behavior, but I would prefer to separate the `generate` and `build` steps. I believe that running the generate script every time you want to build slows down the development process. What do you think?

Reply:
@Garma00 you right, but the goal of this change is to update the toolchain without changing its behavior. We will refactor it in the future, removing unnecessary steps and using better tools (for example `esbuild` instead of `tsc` for transpilation).

Reply:
(`yarn@4` does not support lifecycle scripts, so I had to merge `prebuild` and `build`)